Traceback (most recent call last):
  File "/vast/home/ajherman/continualEP/main.py", line 2, in <module>
    import torchvision.datasets as datasets
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "/vast/home/ajherman/continualEP/main.py", line 2, in <module>
    import torchvision.datasets as datasets
ModuleNotFoundError: No module named 'torchvision'

Inactive Modules:
  1) miniconda3


Activating Modules:
  1) miniconda3/py39_4.12.0

Traceback (most recent call last):
  File "/vast/home/ajherman/continualEP/main.py", line 2, in <module>
    import torchvision.datasets as datasets
ModuleNotFoundError: No module named 'torchvision'

Inactive Modules:
  1) miniconda3


Activating Modules:
  1) miniconda3/py39_4.12.0

Traceback (most recent call last):
  File "/vast/home/ajherman/continualEP/main.py", line 2, in <module>
    import torchvision.datasets as datasets
ModuleNotFoundError: No module named 'torchvision'

Inactive Modules:
  1) miniconda3


Activating Modules:
  1) miniconda3/py39_4.12.0

Traceback (most recent call last):
  File "/vast/home/ajherman/continualEP/main.py", line 2, in <module>
    import torchvision.datasets as datasets
ModuleNotFoundError: No module named 'torchvision'
